crates/biome_css_formatter/tests/specs/css/pseudo/pseudo_class_function_value_list.css.snap

```diff
@@ -0,0 +1,54 @@
+---
+source: crates/biome_formatter_test/src/snapshot_builder.rs
+info: css/pseudo/pseudo_class_function_value_list.css
+---
+
+# Input
+
+```css
+:lang(de, fr) {}
+:lang(
+    de,
+    
+    fr
+    ) {}
+
+:lang(
+    de
+    ) {}
+:lang(
+    de, fr, en, es, hi, pt
+) {}
+```
+
+
+=============================
+
+# Outputs
+
+## Output 1
+
+-----
+Indent style: Tab
+Indent width: 2
+Line ending: LF
+Line width: 80
+-----
+
+```css
+:lang(de, fr) {
+}
+:lang(
+		de,
+
+		fr
+	) {
+}
+
+:lang(de) {
+}
+:lang(de, fr, en, es, hi, pt) {
+}
```

Comment:
https://prettier.io/playground/#N4Igxg9gdgLgprEAuZAbAhlA5gCgCZwA0ABAGYBOAlMcADpQC+9SG2O9tMnBhHUnnChxjU6jPi0y4Co+kyiS2PMuRII1AZxIALAJYkADiJpyQhEBCO7oG5KHTlyEAO4AFBwlsp0qZ+gCetuYARuToYADWcDAAyugAtnAAMrpQcMikPhpEIKHhUbEG4alYyDDkAK452fG6ZZU5cAAeBnDkuomwPgAqbVAOunBemajZ5holqHAAihUQ8BlZOQBWGk0xkzNzC0gjYyAAjttwrk4GXiDoGgC0aXAEeGYg5ei6qCUAwhDx8ejIl6hUE8JtgpgBBGDlXTBCrwVxtFJpRajHLaGDxVAAdT08A0RTAcBinl0MF0ADcSf5-mANEEQGSqgBJKAEWAxMDtIxglkxGD+KbI-YGJzZTFhAz-YVDNpk9LmVLZcgwU7oLC-QU5IrkRXU2lPYWpGCY3R4GDaZAADgADOZyHAjro7Sq1X9dktzDB0MFjabzUgAEzmCrZbpe4bukBweLBe4PJJSCqquAAMQg5F+kJK-3QsIgIAYDCAA

Reply:
Same as the above here, once the selector list stops trying to respect the newlines from the input, this should flatten out and match Prettier.